libs/ui-watt/src/lib/components/shell/nav-list/watt-nav-list.component.spec.ts

```diff
@@ -161,13 +161,13 @@ describe(WattNavListModule.name, () => {
     });
   });
 
-  describe('Verify links added with href attribute opens in expected windows', () => {
+  describe(`${WattNavListModule.name} - Verify links added with href attribute opens in expected windows`, () => {
```

Comment:
suggestion: Revert this change.
```suggestion
  describe('Verify links added with href attribute opens in expected windows', () => {
```